lib/healthcheck/government_data_check.rb

```diff
@@ -0,0 +1,35 @@
+module Healthcheck
+  class GovernmentDataCheck
+    attr_reader :government_repo
+    def initialize
+      @government_repo = BulkData::GovernmentRepository.new
+    end
+
+    def name
+      :government_data_check
+    end
+
+    def status
+      return GovukHealthcheck::CRITICAL unless government_repo.cache_populated?
+      return GovukHealthcheck::WARNING if government_repo.cache_age > 6.hours
+
+      GovukHealthcheck::OK
+    end
+
+    def message
+      return "No government data availible" if status == GovukHealthcheck::CRITICAL
+      return warning_details_content if status == GovukHealthcheck:: WARNING
```

Comment:
This last return is redundant

Reply:
Yeah, i put it in there to keep the linter happy without a new line but it does look a bit nicer with it, have adjusted